kernel/univ.ml

```diff
@@ -499,6 +500,8 @@ struct
 
   let smartmap = List.smartmap
 
+  let map f x =
+    List.fold_right (fun l acc -> (f l) :: acc) x []
```

Comment:
Why not List.map?

Reply:
Indeed, why not :) Fixed

Reply:
Weren't you trying to specify the order?

Reply:
The order of application of [f] in case of side-effects? No :)